src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,159 @@
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+
+function Pagination({ pageContext, type, path }) {
+  let { currentPage, totalPages, minPage, maxPage } = pageContext
```

Comment:
Why are we not destructuring `formData` & `formResponse`?

Reply:
donezo